backend/src/test/java/de/neuefische/backend/controller/ControllerZooFooTest.java

```diff
@@ -25,4 +34,113 @@ void getAllAnimals_returnAllAnimalsAsList_andStatusCode200() throws Exception {
 
     }
 
+    @DirtiesContext
+    @Test
+    void postNewAnimal_expectSuccessfulPost() throws Exception {
+        String actual = mockMvc.perform(
+                        post("/api/animal")
+                                .contentType(MediaType.APPLICATION_JSON)
+                                .content("""
+                                        {
+                                                       "id": "2",
+                                                       "species":"Elephant",
+                                                        "food":"Gras",
+                                                        "foodAmount":10,
+                                                        "dayToFeed":"Monday",
+                                                        "numberOfAnimals":2,
+                                                        "animalKeeper":"Richard",
+                                                        "pictureOfAnimal":""
+                                        }
+                                        """)
+                )
+                .andExpect(status().isOk())
+                .andExpect(content().json("""
+                                        {
+                                                       "id": "2",
+                                                       "species":"Elephant",
+                                                        "food":"Gras",
+                                                        "foodAmount":10,
+                                                        "dayToFeed":"Monday",
+                                                        "numberOfAnimals":2,
+                                                        "animalKeeper":"Richard",
+                                                        "pictureOfAnimal":""
+                                        }
+                        """))
+                .andReturn()
+                .getResponse()
+                .getContentAsString();
+
+        Animal actualAnimal = objectMapper.readValue(actual, Animal.class);
+        assertThat(actualAnimal.getId())
+                .isNotBlank();
```

Comment:
no need for Object mapper. U just checked the id in the expect part content().json